.dockerignore

```diff
@@ -0,0 +1,28 @@
+
+*.pyc
+*.pyo
+*.mo
+*.db
+*.css.map
+*.egg-info
+*.sql.gz
+.cache
+.project
+.idea
+.pydevproject
+.idea/workspace.xml
+.DS_Store
+.git/
+.sass-cache
+.vagrant/
+__pycache__
+dist
+docs
+env
+logs
+src/{{ project_name }}/settings/local.py
+src/node_modules
+web/media
+web/static/CACHE
+stats
+Dockerfile
```

Comment:
please, add a blank for the end of the file

Reply:
i forget about solution from tutorial, and later just google that case(from commit)... i tried it and it works fine, but i can re-do